acestep/ui/gradio/events/training/lora_training.py

```
@@ -0,0 +1,304 @@
+        import shutil
+
+        parent_dir = os.path.dirname(safe_export) or "."
+        os.makedirs(parent_dir, exist_ok=True)
+
+        if os.path.exists(safe_export):
+            shutil.rmtree(safe_export)
+
+        shutil.copytree(source_path, safe_export)
+        return t("training.lora_exported", path=safe_export)
+
+    except Exception as e:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)